config/locales/client.en.yml

```diff
@@ -47,9 +47,9 @@ en:
         translation: "Translation"
```

Comment:
@keegangeorge The formatting changes in this file are useful, but we need to seperate them as they're not really part of this PR. Could you make a seperate one with them? I know it's a bit annoying, but we should try and follow our PR rules here.